### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by JowelTisso**
This PR is for Authentication screen.

Note:
1. There are two screens login and signup.
2. They most share similar design and code.
3. There is one common css for these two screen

### CODE REVIEW

File: screens/authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.toggle-icon-container {
+  position: relative;
+}
+
+.toggle-icon {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
```
Comment (Tarandeep-s1ngh) [inline screens/authentication/auth.css]: better to use percentage for absolute positions

File: screens/authentication/login/login.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cart</title>
```
Comment (Tarandeep-s1ngh) [inline screens/authentication/login/login.html]: Update the title

File: screens/authentication/signup/signup.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cart</title>
```
Comment (Tarandeep-s1ngh) [inline screens/authentication/signup/signup.html]: Update the title for this also

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: screens/authentication/auth.css (hunk 1 above)
Comment (JowelTisso) [reply screens/authentication/auth.css]: > better to use percentage for absolute positions

Changes made, Thanks for the feedback.

File: screens/authentication/login/login.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    <link
+      rel="shortcut icon"
+      href="../../../assets/logo.png"
```
Comment (znjs) [inline screens/authentication/login/login.html]: if possible use absolute paths

**Review by znjs — APPROVED** [review]

File: screens/authentication/login/login.html (hunk 4 above)
Comment (JowelTisso) [reply screens/authentication/login/login.html]: Ok, got your point, Thanks for the feedback.

**Pull request merged**